### DISCUSSION TITLE
How to call different completitions on different keybindings.

### DISCUSSION THREAD

**Discussion by Shuraken007** [neoclide/coc.nvim#4784] [General]
Hello. Is it possible to bind smth like:

bind1 - show only functions for complete
bind2 - show only classes
bind3 - show current file variables
bind4 - show all files variables
bind5 - show only snippets

I want to use most of time smth like tab_nine.
And run some special binds to show exactly part that I need, when AI is not enough smart.

**Comment by fannheyward** [comment]
No such feature.
